modules/ossm-about-adding-namespace.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: CONCEPT
+[id="ossm-about-adding-namespace_{context}"]
+= About adding projects to a service mesh
+
+A project contains services; however, the services are not available to the mesh unless you add the project to the service mesh resource.
+
+[NOTE]
+====
+In OpenShift Container Platform, a project is essentially a Kubernetes namespace with additional annotations, such as the range of user IDs that can be used in the project.
+====
+
+You can add projects to an existing service mesh using either the web console or the CLI. There are two methods to add a project to a mesh:
+
+* Specifying a project in the `ServiceMeshMemberRoll` resource.
+
+* Creating a `ServiceMeshMember` resource in a project.
+
+.ServiceMeshMemberRoll method
+
+This is the simplest way to add a project to a service mesh. To add a project, specify the project namespace where your applications reside in the `spec.members` field of the `ServiceMeshMemberRoll` resource.
+
+[NOTE]
+====
+Adding projects using this method requires a user to have mesh admin privileges.
+====
+
+The items listed in the `ServiceMeshMemberRoll` resource are the applications and workflows that are managed by the `ServiceMeshControlPlane` resource. The control plane, which includes the {SMProductShortName} Operators, Istiod, and `ServiceMeshControlPlane`, and the data plane, which includes applications and the Envoy proxy, must be in separate namespaces.
+
+* If you already have an application, workload, or service to add to a `ServiceMeshMemberRoll` resource, see the instructions for adding or removing projects from the service mesh using the `ServiceMeshMemberRoll` resource with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-add-project-member-roll-recourse-console_ossm-create-mesh[web console] or with the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-add-project-member-roll-resource-cli_ossm-create-mesh[CLI]. 
+
+* Alternatively, to install a sample application called Bookinfo and add it to a `ServiceMeshMemberRoll` resource, skip to the tutorial for installing the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo example application] to see how an application works in {SMProductName}.
```

Comment:
BTW, every time I read this, I read it as "skip the tutorial" instead of "skip *to* the tutorial". You might want to rephrase.

Reply:
+1

Reply:
```suggestion
* Alternatively, to install a sample application called Bookinfo and add it to a `ServiceMeshMemberRoll` resource, see the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo example application] tutorial.
```